tests/lib.rs

```diff
@@ -6,6 +6,10 @@ use log::LogLevel;
 use wikilib::Wiki;
 
 use std::path::Path;
+use std::fs;
+
+static NON_EXISTING_DIR: &'static str = "_should_not_exist_";
```

Comment:
This can be now ellided since Rust 1.17:
```
static NON_EXISTING_DIR: &str = "_should_not_exist_";
```

Maybe you can search/replace threw the whole project and replace all of them?